src/opencmiss_iron.f90

```diff
@@ -3960,7 +3974,8 @@ INTEGER(INTG), PARAMETER :: CMFE_FIELD_NODE_BASED_INTERPOLATION = FIELD_NODE_BAS
     & CMFE_FIELD_PREVIOUS_ACCELERATION_SET_TYPE, &
     & CMFE_FIELD_MEAN_PREDICTED_ACCELERATION_SET_TYPE, CMFE_FIELD_PRESSURE_VALUES_SET_TYPE, &
     & CMFE_FIELD_PREVIOUS_PRESSURE_SET_TYPE, &
-    & CMFE_FIELD_IMPERMEABLE_FLAG_VALUES_SET_TYPE,CMFE_FIELD_INTEGRATED_NEUMANN_SET_TYPE
+    & CMFE_FIELD_IMPERMEABLE_FLAG_VALUES_SET_TYPE,CMFE_FIELD_INTEGRATED_NEUMANN_SET_TYPE, &
+    & CMFE_FIELD_MESH_DISPLACEMENT_SET_TYPE,CMFE_FIELD_MESH_VELOCITY_SET_TYPE
```

Comment:
Yes, apologies, I thought the diff showed it as an addition which isn't the case.